src/main/jbake/content/jms-concepts005.adoc

```diff
@@ -243,39 +243,39 @@ and to consume messages.
 
 Because a synchronous receive with no specified timeout ties up server
 resources, this mechanism usually is not the best application design for
-a web or EJB component. Instead, use a synchronous receive that
+a web or Jakarta Enterprise Beans component. Instead, use a synchronous receive that
 specifies a timeout value, or use a message-driven bean to receive
 messages asynchronously. For details about synchronous receives, see
-link:jms-concepts003.html#BNCEP[JMS Message Consumers].
+link:jms-concepts003.html#BNCEP[Jakarta Messaging Message Consumers].
 
-Using the JMS API in a Jakarta EE component is in many ways similar to
+Using Jakarta Messaging in a Jakarta EE component is in many ways similar to
 using it in an application client. The main differences are the areas of
 resource management and transactions.
 
 [[BNCGO]][[managing-jms-resources-in-web-and-ejb-components]]
 
-Managing JMS Resources in Web and EJB Components
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Managing Jakarta Messaging Resources in Web and Jakarta Enterprise Beans Components
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Ditto.

(Did you write a script to fix the underlines?  Only the "^" underlines are wrong.)

Reply:
Let's say I'm using visual recognition :). Perhaps I should try to write some scripts for this.

I think I have now fixed all. Thanks for the review.

Reply:
I guess it's time for a Maven plugin...  :-)

Reply:
Hopefully we can eventually get rid those underlines...

Reply:
Yes, there's a different style for AsciiDoc section titles that doesn't use underlines, but...
1. The Maven plugin that generates the Table of Contents needs to be updated to understand it.
2. The title style has subtle implications on some other AsciiDoc syntax that needs to be considered.

Reply:
I will give it a try now we have completed the very first step. Btw thanks a lot for you thorough reviews!